src/components/Homepage.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import TourCard from './TourCard';
+
+function Homepage() {
+  return (
+    <section className="mx-auto w-screen">
+      <div className="mx-auto w-full flex flex-col p-10">
+        <h1 className="mx-auto font-bold text-2xl text-black">Our Tours</h1>
+        <p className="mx-auto text-gray-dark">Please select one of our tours</p>
+      </div>
+      <div className="w-full flex flex-col">
+        <TourCard
+          tourName="Eiffel Tower"
+          tourImage="https://images.unsplash.com/photo-1511739001486-6bfe10ce785f?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+          tourLocale="Paris"
+        />
+      </div>
+    </section>
```

Comment:
- [ ] [OPTIONAL] I think that it would be nice if you add a gap between the Tour cards because it will improve the cards display. You can use the Tailwind's gap class because it is a flex container :slightly_smiling_face: 

![image](https://user-images.githubusercontent.com/61767582/178073130-9e3da517-66a3-4de9-9491-c6c419f823c2.png)
The social media links are close to the next card.